tasks/update_go.py

```diff
@@ -11,53 +11,14 @@
 
 # list all platforms used in the repo
 # store the platform names in the format expected by Go
-# the variables in the Dockerfiles should be stored in uppercased variables
 # see https://go.dev/dl/ for the full list of platforms
-PLATFORMS: Set[Platform] = {
+PLATFORMS: List[Platform] = {
```

Comment:
Is it working to type it as a `List` and define it with `{ }`. Should we change it to `[ ]` ?

Reply:
It was disabled :smile: